aws/modules/infrastructure_modules/container_registry/variables.tf

```diff
@@ -64,3 +66,9 @@ variable "pull_through_cache_accounts" {
   description = "A default list of accounts for the Pull Through Cache if not configured in the `pull_through_cache_setup`. Defaults to the calling account root"
   default     = []
 }
+
+variable "repository_lifecycle_policy" {
+  type        = any
```

Comment:
This possibly should be a strongly typed object and if null is passed use the default?